web/packages/teleport/src/config.ts

```diff
@@ -492,6 +497,22 @@ const cfg = {
     });
   },
 
+  getLocksRoute(clusterId: string) {
+    return generatePath(cfg.routes.locks, { clusterId });
+  },
+
+  getNewLocksRoute(clusterId: string) {
+    return generatePath(cfg.routes.newLock, { clusterId });
+  },
+
+  getLocksUrl(clusterId: string) {
+    return generatePath(cfg.api.locksPath, { clusterId });
+  },
+
+  getLocksUrlWithUUID(clusterId: string, uuid: string) {
```

Comment:
Yeah, see how we used `Url` above instead of `URL`? I think the Typescript convention we just is just to capitalize the first letter of an initialism (unlike in Go where we capitalize all of them)